### DISCUSSION TITLE
Help with Dynamic / on the Fly masks

### DISCUSSION THREAD

**Discussion by ttorquati** [uNmAnNeR/imaskjs#580] [Q&A]
So I am trying to set two masks depending on what length the user inputs, however, it always choose the first one no matter the order.

In the example below, if the user inputs is 11 characters or less, should apply the first mask, if more than 11 then should apply the second one.

Should I got for a more manually approach, or can I achieve somehow using like this?

```javascript
this.iMask = IMask(this.inputElement, 
{ 
    mask: [
        {
            mask: '000.000.000-00',
        },
        {
            mask: '00.000.000/0000-00'
        }
    ]
});
```

I am not sure if that is the usage of this mask.

Thanks Thanks

**Comment by marcosrocha85** [comment]
I think it's the same issue I'm having at #915 for phone number. I'll keep you updated if I find something.
